templates/.reek.yml

```diff
@@ -13,6 +13,14 @@ directories:
     "app/helpers":
       UtilityFunction:
         enabled: false
+      FeatureEnvy:
+        enabled: false
+    "app/jobs":
+      UtilityFunction:
+        enabled: false
+    "app/services":
```

Comment:
looked at some examples. it's a common pattern to use this `run` and there it would for example complain